source/Commands/CommandObjectExpression.cpp

```diff
@@ -398,9 +398,50 @@ bool CommandObjectExpression::EvaluateExpression(const char *expr,
       options.SetTimeout(std::chrono::microseconds(m_command_options.timeout));
     else
       options.SetTimeout(llvm::None);
-
-    ExpressionResults success = target->EvaluateExpression(
-        expr, frame, result_valobj_sp, options, &m_fixed_expression);
+    
+    // Swift does not have an IR interpreter, so expression evaluation for
+    // simple variable access is quite slow.  Accelerate that by first seeing
+    // if the whole expression is a simple identifier, and if so look it up
+    // as a local variable.
+    // Note, Swift's type resolution rules are complex enough that even figuring
+    // out whether the dynamic and static versions of foo.bar are the same is
+    // tricky, so we only handle single identifiers.
+    // Also, we don't do this if we are using the REPL, top level code, 
+    // or if the PlaygroundTransform is enabled.
+    ExpressionResults success = eExpressionSetupError;
+    if (target->GetUseFrameVarToAccelerateExpr()
```

Comment:
Can you split the fast path in a different function (and clang-format for consistency) ?